### DISCUSSION TITLE
make horn unreachable when using the /horn command

### DISCUSSION THREAD

**Discussion by zx3800** [winstxnhdw/lc-hax#211] [Ideas]
people can stop the noise by stocking the thing, so place it far from the ship or out of bounds so the madness never stops

**Comment by winstxnhdw** [comment]
They shouldn't be able to stop the horn.. Make sure you set the duration to a really long one.

**Reply by zx3800** [reply]
i set it to 9999, but by putting away the horn in object placement mode, it stops it

**Reply by winstxnhdw** [reply]
Ah, you mean if they put it in the storage. I'll do something.

**Reply by winstxnhdw** [reply]
Merged. Try latest.

**Reply by zx3800** [reply]
Well, now it spawns it, but it doesnt activate the horn

**Reply by winstxnhdw** [reply]
Yeah, sorry about that, seems like the new update broke stuff.

**Reply by winstxnhdw** [reply]
Hi, please read the recent announcement on this issue.
